br/pkg/backup/push.go

```diff
@@ -22,6 +23,11 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	// from PD: https://github.com/tikv/pd/blob/c40e319f50822678cda71ae62ee2fd70a9cac010/pkg/core/store.go#L523
+	storeDisconnectionDuration = 20 * time.Second
```

Comment:
what if the value changed?

And can user change store heartbeat duration manually?

Reply:
How can we configure this value? If you afraid this value is too pessimistic, perhaps we can make it longer. (say, 5 mins)